tests/test_admin_actions.py

```diff
@@ -64,10 +68,11 @@ def test_export_to_csv_action(self):
 
         content = list(i.decode('utf-8') for i in response.streaming_content)
         self.assertEqual(len(content), TestModel.objects.count() + 1)
-        self.assertRegexpMatches(content[0], r'^ID,title,tags_count')
-        self.assertRegexpMatches(content[1], r'^1,TEST ITEM 1,0\r\n')
-        self.assertRegexpMatches(content[2], r'^2,TEST ITEM 2,0\r\n')
+        self.assertRegex(content[0], r'^ID,title,tags_count')
+        self.assertRegex(content[1], r'^1,TEST ITEM 1,0\r\n')
+        self.assertRegex(content[2], r'^2,TEST ITEM 2,0\r\n')
 
+    @unittest.skip("error (where is custom export configured?)")
```

Comment:
The underlying problem here is probably that the form parameters which the Django admin interface expects has probably changed: it's generating a 302 redirect to the requested URL so the test code must be missing something basic.

```python
(Pdb) p response
<HttpResponseRedirect status_code=302, "text/html; charset=utf-8", url="/admin/tests/testmodel/">
(Pdb) p changelist_url
'/admin/tests/testmodel/'
```

Reply:
Hi both! @rlskoeser asked me to take a look at this.

The error message Django sends back here is "No action selected." You can check this with:
```py
from django.contrib.messages import get_messages
messages = [m.message for m in list(get_messages(response.wsgi_request))]
```
I notice that there's no action actually named `custom_export_to_csv_action` so maybe this is the intended behavior? While that is the same error message you get on form validation errors, I don't see any difference in the form params—the underlying Django code hasn't actually changed ([Django 3.2.x](https://github.com/django/django/blob/3591e1c1acbd7c13174275367c3fdf012cb0413b/django/contrib/admin/options.py#L1384-L1420), [Django 5](https://github.com/django/django/blob/01a4d8a3c741b3129d481ef3515084a199d21222/django/contrib/admin/options.py#L1640-L1678)), and I can't get this test to succeed under Django 3.2 either.

It seems correct to redirect with this error message unless an admin action with that exact name was registered, unless I am missing something.

Reply:
Thanks for looking into this, @blms. 

@acdha let us know how you'd like to resolve this.